src/rpc/blockchain.cpp

```diff
@@ -485,6 +485,89 @@ UniValue getblockdeltas(const UniValue& params, bool fHelp)
     return blockToDeltasJSON(block, pblockindex);
 }
 
+// insightexplorer
+UniValue getblockhashes(const UniValue& params, bool fHelp)
+{
+    std::string enableArg = "insightexplorer";
+    bool fEnableGetBlockHashes = fExperimentalMode && fInsightExplorer;
+    std::string disabledMsg = "";
+    if (!fEnableGetBlockHashes) {
+        disabledMsg = experimentalDisabledHelpMsg("getblockhashes", enableArg);
+    }
+    if (fHelp || params.size() < 2)
+        throw runtime_error(
+            "getblockhashes high low ( {\"noOrphans\": true|false, \"logicalTimes\": true|false} )\n"
```

Comment:
I thought the same thing, but ... compatibility! It will be nice if and when we control that insight nodejs code, and we can make the interfaces more sane, although we'd have to consider if there are other explorers out there that we still wouldn't control that will want to use our `zcashd`.

Also, I don't like the way the defaults work. IIRC, if either is zero, then the other is ignored, and the method returns the full range. It would be nice if `high` could be zero to mean from `low` to the current best height.